common/rpm/update-config-files.sh

```diff
@@ -5,45 +5,45 @@
 set -exo pipefail
 
 ensure() {
-  if [[ ! -f $1 ]]; then
-      echo "File not found:" $1
+  if [[ ! -f "$1" ]]; then
+      echo "File not found:" "$1"
       exit 1
   fi
-  if grep ^$2[[:blank:]].*= $1 > /dev/null
+  if grep "^$2[[:blank:]].*=" "$1" > /dev/null
   then
-    sed -i "s;^$2[[:blank:]]=.*;$2 = $3;" $1
+    sed -i "s;^$2[[:blank:]]=.*;$2 = $3;" "$1"
   else
-    if grep ^\#.*$2[[:blank:]].*= $1 > /dev/null
+    if grep "^\#.*$2[[:blank:]].*=" "$1" > /dev/null
     then
       sed -i "/^#.*$2[[:blank:]].*=/a \
-$2 = $3" $1
+$2 = $3" "$1"
     else
-      echo "$2 = $3" >> $1
+      echo "$2 = $3" >> "$1"
     fi
   fi
 }
 
 # Common options enabled across all fedora, centos, rhel
 # TBD: Can these be enabled by default upstream?
-ensure registries.conf              short-name-mode     \"enforcing\"
+ensure image/registries.conf              short-name-mode     \"enforcing\"
 
-ensure storage.conf                 driver              \"overlay\"
-ensure storage.conf                 mountopt            \"nodev,metacopy=on\"
+ensure storage/storage.conf               driver              \"overlay\"
+ensure storage/storage.conf               mountopt            \"nodev,metacopy=on\"
 
-ensure pkg/config/containers.conf   runtime             \"crun\"
-ensure pkg/config/containers.conf   log_driver          \"journald\"
+ensure common/pkg/config/containers.conf  runtime             \"crun\"
+ensure common/pkg/config/containers.conf  log_driver          \"journald\"
```

Comment:
There was some … discussion in #4 about how packaging happens. Does this risk interfering with downstream RPMs?